.github/workflows/ci.yml

```diff
@@ -32,13 +32,13 @@ jobs:
     steps:
       # Cancel any previous runs for the same branch that are still running.
       - name: 'Cancel previous runs'
-        uses: styfle/cancel-workflow-action@85880fa0301c86cca9da44039ee3bb12d3bedbfa
+        uses: styfle/cancel-workflow-action@85880fa0301c86cca9da44039ee3bb12d3bedbfa # 0.12.1
```

Comment:
The version number here has no leading `v` because the Git tags in that repository don't have it either. Dependabot [should be able to update this as well](https://github.com/dependabot/dependabot-core/blob/795dafedf31666de6e91b078b799fd7f2517478b/github_actions/spec/dependabot/github_actions/file_updater_spec.rb#L384).